packages/fsbazaarvoice/src/BazaarvoiceDataSource.ts

```diff
@@ -31,18 +32,26 @@ export class BazaarvoiceDataSource extends AbstractReviewDataSource implements R
 
   async fetchReviewDetails(query: ReviewTypes.ReviewQuery): Promise<ReviewTypes.ReviewDetails[]> {
     const id = Array.isArray(query.ids) ? query.ids[0] : query.ids;
+    const filter = query.filter || `ProductId:${id}`;
+
     const params: BazaarvoiceReviewRequest = {
-      Filter: `ProductId:${id}`,
+      Filter: filter,
       Include: 'Products',
       Stats: 'Reviews',
-      Limit: query.limit || 10
+      Limit: query.limit || 10,
+      ...(query.sort ? {Sort: query.sort} : {})
```

Comment:
Why not just make it `Sort: query.sort`? Is it shoving in "Sort=undefined" that way?

Reply:
I wasn't sure if it would still add an undefined parameter or not.

I just checked an qs indeed omits them, so I can change this.